wukong/request.py

```diff
@@ -27,105 +31,135 @@ class SolrRequest(object):
     """
     Handle requests to SOLR and response from SOLR
     """
+
+    REFRESH_FREQUENCY = 2
```

Comment:
Can you put a comment with the units here?
```py
REFRESH_FREQUENCY = 2  # years
```
or something

Reply:
Done